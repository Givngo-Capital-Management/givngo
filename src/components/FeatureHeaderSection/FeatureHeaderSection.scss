.section {
  display: flex;
  padding-top: 100px;
  padding-bottom: 50px;
  -webkit-box-align: center;
  align-items: center;

  .div-wrap {
    width: 600px;
    margin-right: auto;
    margin-left: auto;
    border-radius: 1px;
  }

  .heading {
    margin-top: 0px;
    margin-bottom: 20px;
    font-family: sans-serif;
    color: #fa7d00;
    font-size: 72px;
    line-height: 72px;
    font-weight: 700;
    text-align: center;
    letter-spacing: -2px;
  }

  .text-block {
    margin-right: 80px;
    margin-bottom: 30px;
    margin-left: 80px;
    font-size: 24px;
    line-height: 30px;
    text-align: center;
  }
}