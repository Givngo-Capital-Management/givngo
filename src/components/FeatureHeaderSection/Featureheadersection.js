import React from 'react'
import './FeatureHeaderSection.scss'

export default function FeatureHeaderSection() {
  return (
    <div class="section" style={{paddingBottom: "50px"}}>
      <div class="div-wrap">
        <h1 class="heading">Donation creation with givngo</h1>
        <div class="text-block">Powerful tools to increase donations, grow membersm and create awareness.</div>
      </div>
    </div>
  )
}