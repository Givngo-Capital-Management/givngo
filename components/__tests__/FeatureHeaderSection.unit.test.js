import React from 'react';
import { shallow } from 'enzyme';
import FeatureHeaderSection from '../Features/FeatureHeaderSection';

describe('FeatureHeaderSection', () => {
  test('renders', () => {
    const wrapper = shallow(<FeatureHeaderSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
