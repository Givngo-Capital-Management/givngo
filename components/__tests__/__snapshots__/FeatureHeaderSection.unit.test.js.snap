// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureHeaderSection renders 1`] = `
<div
  style={
    Object {
      "paddingBottom": "50px",
    }
  }
>
  <div>
    <h1>
      Donation creation with givngo
    </h1>
    <div>
      Powerful tools to increase donations, grow membersm and create awareness.
    </div>
  </div>
</div>
`;
